numba/cuda/simulator/cudadrv/devicearray.py

```diff
@@ -188,6 +188,7 @@ def check_array_compatibility(ary1, ary2):
 
 
 def to_device(ary, stream=0, copy=True, to=None):
+    ary = np.array(ary, copy=False, subok=True)
```

Comment:
note to self. replicates https://github.com/numba/numba/blob/9a3f13020475ea945c7f0359db3e3350484710d6/numba/cuda/cudadrv/devicearray.py#L699 semantics